src/external_connections/next_gen_io.py

```diff
@@ -0,0 +1,61 @@
+#Utility I/O functions for the next generation water model network.
+#The read_qlat and replace_downstreams functions are copies from nhd_io.py
+#and are placed here for now, so that this program can run without importing that module.
+import xarray as xr
+import pandas as pd
+import geopandas as gpd
+import os
+
+
+def read_ngen_network_csv(path):
+    return pd.read_csv(path, index_col=0)
+
+
+def read_ngen_network_geojson(path):
+    return gpd.read_file(path)
+
+
+def read_ngen_network_json(path):
+    return pd.read_json(path)
+
+
+#Convert catchment lateral flows to format that can be processed by compute_network
+def read_catchment_lateral_flows(path):
+    ql = []
+    qlats_df = pd.DataFrame({'A' : []})
```

Comment:
Is this needed?

Reply:
This function is needed for now because the qlats that next gen produces are in a different format than the qlats that are currently fed in. I think that the next gen lateral flow format is tentative and could change.

Reply:
I was referring to that specific line where qlats_df is created. It isn't used and discarded before this function returns.

Reply:
Ok good point thanks. Removing this.